projector-plugin/src/main/resources/META-INF/versions.txt

```diff
@@ -0,0 +1 @@
+agentVersion=1.0.0
```

Comment:
Is there any reason why we should include generated files in the repo? FMPOV, we should ignore them.

Reply:
Agreed.